### PR TITLE
Please review branch feature/ZEN-22889

### PR DESCRIPTION
**Pull request by romanbakaleyko**
Cherry-picked 39c69d1db77cd8e8281e55a5d2d08d07befe5c0b

Pull request created by zendev port

### CODE REVIEW

**Comment by olst** [comment]
retest this

**Comment by romanbakaleyko** [comment]
retest this

File: bin/zenossdbpack [hunk 1]
```diff
@@ -101,6 +86,8 @@ if __name__ == "__main__":
         "-d", "--days", dest="days", type=float, default=0.5, # 12 hours
         help="Days to keep unreferenced objects (default: %(default)s). "
         "NOTE: To help prevent PKEs, zenossdbpack will not accept --days below 0.5")
+    # -e option is no longer used. Left for compatibility purposes.
+    # Many customers have cron jobs passing -e session
     parser.add_argument(
         "-e", "--exclude-db", dest="excludeDB",
         choices=['session', 'zodb'],
```
Comment (glujan) [inline bin/zenossdbpack]: I think you should change help text for this argument to say it has no effect - just what you've done in comment above for developers

**Comment by glujan** [comment]
retest this please

**Comment by romanbakaleyko** [comment]
retest this

**Comment by romanbakaleyko** [comment]
retest this

**Comment by romanbakaleyko** [comment]
retest

**Comment by romanbakaleyko** [comment]
retest this

**Pull request merged**